File: src/main/java/com/zzz/dao/UserRoleRepository.java
package com.zzz.dao;

import com.zzz.model.po.UserRolePo;
import org.springframework.data.repository.Repository;

import java.util.List;

/**
 * @author hushengjun
 * @date 2018/3/8
 */
public interface UserRoleRepository extends Repository<UserRolePo, Integer> {

    List<UserRolePo> findByUser(Integer user);

}

File: src/main/java/com/zzz/dao/UserRoleRepository.kt
package com.zzz.dao

import com.zzz.model.po.UserRolePo
import org.springframework.data.repository.Repository

/**
 * @author hushengjun
 * *
 * @date 2018/3/8
 */
interface UserRoleRepository : Repository<UserRolePo, Int> {

    fun findByUser(user: Int?): List<UserRolePo>

}
